src/modules/job-manager/jobtap.c

```diff
@@ -481,11 +482,14 @@ static void jobtap_handle_load_req (struct job_manager *ctx,
 
     /*  Make plugin aware of all active jobs via job.new callback
      */
-    job = zhashx_first (ctx->active_jobs);
+    jobs = zhashx_values (ctx->active_jobs);
```

Comment:
Check `zhashx_values()` for NULL return (verified in the czmq source that it can do that).